src/presentation/pages/Preferences/ServiceCenter/index.tsx

```diff
@@ -32,6 +34,9 @@ function ServiceCenterPage() {
     if (selectedItemID === undefined && categories !== undefined) {
       setSelectedItemID(categories[0].id);
     }
+    if (categoryState && categories !== undefined) {
```

Comment:
categoryState가 뭘 의미하는지 다소 명확하지 않은 것 같은데 state를 { isError: boolean } 같은 형태로 넘겨줘서 아래처럼 수정하는 것이 어떨까요?

```suggestion
    if (location.state.isError && categories !== undefined) {
```

Reply:
저도 동의합니다 리뷰 반영해서 수정했습니다 !!

Reply:
참고로 같이 알면 좋은 내용 남깁니다!

`location.state.isError`를 바로 사용하려고 했더니 type unknown 오류가 뜨더라구요
무엇이 원인인지 찾아봤더니, react-router-dom이 v6로 업그레이드되면서
location의 state 속성이 무조건 unknown으로 인식된다고 합니다.
그래서 아래 링크들 참고해서 타입 에러 해결해주었습니다 !!

[참고 1](https://github.com/reach/router/issues/414)
[참고 2](https://velog.io/@mael1657/TypeScript-react-router-v6-type-unknown-%EC%98%A4%EB%A5%98)